tools/tracer/tools/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+add_subdirectory(configure)
+
+set (TRACER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/..")

```

Comment:
why this is located in tools directory? why not one layer higher?

Reply:
fixed